### PR TITLE
Update pango.rb

### PR DESCRIPTION
**Pull request by lovedheart**
Update pango build to avoid compilation error to build gtk+

Works properly:
- [ ] x86_64
- [x] armv7l

### CODE REVIEW

File: packages/pango.rb [hunk 1]
```diff
@@ -3,35 +3,39 @@
...
+#     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/pango-1.41-chromeos-armv7l.tar.xz',
+#       i686: 'https://dl.bintray.com/chromebrew/chromebrew/pango-1.41-chromeos-i686.tar.xz',
+#     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/pango-1.41-chromeos-x86_64.tar.xz',
+#  })
+#  binary_sha256 ({
+#    aarch64: '67f8f261f2a069225b2858fe6dc622a0a1eb4a04e1172be6e3bb16ce09c21785',
+#     armv7l: '67f8f261f2a069225b2858fe6dc622a0a1eb4a04e1172be6e3bb16ce09c21785',
+#       i686: '032ddf476182d20ee2db104abae1bf918db4d14ef931e8011231def15a9eaffc',
+#     x86_64: '88e9f5e2c23542edbb724011fb1bd9190770de53fcf0c2a382ea4e26ed5579b3',
+#  })
```
Comment (uberhacker) [inline packages/pango.rb]: If you could just remove these lines altogether please, that would be better.

File: packages/pango.rb [hunk 2]
```diff
@@ -3,35 +3,39 @@
...
+  depends_on 'python27'                # need six package from python to install Pango
+
 
   def self.build
     system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
     system "make"
   end
 
   def self.install
+    system "pip install six"
```
Comment (uberhacker) [inline packages/pango.rb]: Should this be a separate package?  Also, you will need `--prefix #{CREW_PREFIX}` and `--root #{CREW_DEST_DIR}` so the files can be tracked by the package.

Comment (lovedheart) [reply packages/pango.rb]: Thanks for the tip

**Comment by lovedheart** [comment]
Based on this mod , I built the gtk+ locally

**Pull request merged**